aws/resource_aws_efs_file_system_test.go

```diff
@@ -351,3 +374,33 @@ resource "aws_efs_file_system" "foo-with-performance-mode" {
 	performance_mode = "maxIO"
 }
 `
+
+func testAccAWSEFSFileSystemConfigWithKmsKey(rInt int) string {
+	return fmt.Sprintf(`
+	resource "aws_kms_key" "foo" {
```

Comment:
I actually dont think we need that policy at all, test runs fine without it so I removed it